src/styles/_base.scss

```diff
@@ -6,18 +6,352 @@ body {
+  }
+}
+
+h1--alt, .h1--alt {
+  font-size: 28px;
+  line-height: 28px;
+  font-family: "Lato", sans-serif;
+  font-weight: 700;
+  margin: 0;
+  font-family: "Lato", sans-serif;
+  font-weight: 700;
+}
+
+@media screen and (min-width: 360px) {
+  h1--alt, .h1--alt {
+    font-size: calc(28px + 27 * (100vw - 360px) / 840);
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  h1--alt, .h1--alt {
+    font-size: 55px;
+  }
 }
+
+@media screen and (min-width: 360px) {
+  h1--alt, .h1--alt {
+    line-height: calc(28px + 27 * (100vw - 360px) / 840);
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  h1--alt, .h1--alt {
+    line-height: 55px;
+  }
+}
+
+h2, .h2 {
+  font-size: 23px;
+  line-height: 23px;
+  font-family: "Lato", sans-serif;
+  font-weight: 700;
+}
+
+@media screen and (min-width: 360px) {
+  h2, .h2 {
+    font-size: calc(23px + 5 * (100vw - 360px) / 840);
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  h2, .h2 {
+    font-size: 28px;
+  }
+}
+
+@media screen and (min-width: 360px) {
+  h2, .h2 {
+    line-height: calc(23px + 5 * (100vw - 360px) / 840);
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  h2, .h2 {
+    line-height: 28px;
+  }
+}
+
+h2--alt, .h2--alt {
+  font-size: 23px;
+  line-height: 23px;
+  font-family: "Lato", sans-serif;
+  font-weight: 700;
+  font-family: "Lato", sans-serif;
+  font-weight: 300;
+}
+
+@media screen and (min-width: 360px) {
+  h2--alt, .h2--alt {
+    font-size: calc(23px + 5 * (100vw - 360px) / 840);
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  h2--alt, .h2--alt {
+    font-size: 28px;
+  }
+}
+
+@media screen and (min-width: 360px) {
+  h2--alt, .h2--alt {
+    line-height: calc(23px + 5 * (100vw - 360px) / 840);
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  h2--alt, .h2--alt {
+    line-height: 28px;
+  }
+}
+
+h3, .h3 {
+  font-size: 18px;
+  line-height: 28px;
+  font-family: "Lato", sans-serif;
+  font-weight: 700;
+  line-height: 30px;
+  margin-bottom: 0;
+}
+
+@media screen and (min-width: 360px) {
+  h3, .h3 {
+    font-size: calc(18px + 5 * (100vw - 360px) / 840);
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  h3, .h3 {
+    font-size: 23px;
+  }
+}
+
+@media screen and (min-width: 360px) {
+  h3, .h3 {
+    line-height: calc(28px + 4 * (100vw - 360px) / 840);
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  h3, .h3 {
+    line-height: 32px;
+  }
+}
+
+h3--alt, .h3--alt {
+  font-size: 23px;
+  line-height: 23px;
+  font-family: "Lato", sans-serif;
+  font-weight: 700;
+  font-family: "Lato", sans-serif;
+  font-weight: 300;
+}
+
+@media screen and (min-width: 360px) {
+  h3--alt, .h3--alt {
+    font-size: calc(23px + 5 * (100vw - 360px) / 840);
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  h3--alt, .h3--alt {
+    font-size: 28px;
+  }
+}
+
+@media screen and (min-width: 360px) {
+  h3--alt, .h3--alt {
+    line-height: calc(23px + 5 * (100vw - 360px) / 840);
+  }
+}
+
+@media screen and (min-width: 1200px) {
```

Comment:
Please review the 1200px media query, the header text is cut (i.e. "BIOCLIMATIC VARIAB"). I know we're not supporting mobile, but I don't think 1200px should be an issue for that.

Reply:
Fixed here: https://github.com/Vizzuality/copernicus-forest/pull/33/commits/d76cf8d043b4f4a776994fc7d880fe0791eefec5